guides/source/active_record_multiple_databases.md

```diff
@@ -320,7 +320,39 @@ decide when to swap connections. You can write your own class:
 
 ```ruby
 class MyCookieResolver
```

Comment:
The original documentation is incorrect, this should inherit from the Active Record's default Resolver so you don't need to reimplement all the methods below, you would only need to reimplement the ones that are different.

Then you should delete docs for the following methods:
* `convert_time_to_timestamp`
* `convert_timestamp_to_time`